apps/web/src/lib/components/command-palette/providers/ContextualSearchProvider.ts

```
@@ -0,0 +1,168 @@
+import Calendar from 'lucide-svelte/icons/calendar'
+import FileText from 'lucide-svelte/icons/file-text'
+import Handshake from 'lucide-svelte/icons/handshake'
+import ListOrdered from 'lucide-svelte/icons/list-ordered'
+import Map from 'lucide-svelte/icons/map'
```

Biome (2.3.13)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)